<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AstroPi</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../.designer/assets/AstroPi.png</normaloff>../../../../../.designer/assets/AstroPi.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>0</y>
      <width>331</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ready...</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>590</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;p style=&quot;color:red&quot;&gt;&lt;strong&gt;Please connect your board.&lt;/strong&gt;&lt;/p&gt;</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>321</width>
        <height>581</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>296</width>
         <height>669</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>&lt;h2 align=&quot;center&quot;&gt;POWER&lt;/h2&gt;</string>
          </property>
         </widget>
        </item>
        <item row="24" column="0" colspan="2">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>&lt;h2 align=&quot;center&quot;&gt;SYSTEM&lt;/h2&gt;</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>â†’</string>
          </property>
         </widget>
        </item>
        <item row="28" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>Connect to shell...</string>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <widget class="QSlider" name="horizontalSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="18" column="1">
         <widget class="QComboBox" name="comboBox_3">
          <item>
           <property name="text">
            <string>ON</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OFF</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>Really short (0-5 min)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Short (5-30 min)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Normal (30-60 min)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Long (1-2 hrs)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Really long (whole night?)</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="15" column="1">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Auto</string>
          </property>
         </widget>
        </item>
        <item row="31" column="0" colspan="2">
         <widget class="QLabel" name="label_39">
          <property name="text">
           <string>&lt;p align=&quot;center&quot;&gt;Coming soon!&lt;/p&gt;</string>
          </property>
         </widget>
        </item>
        <item row="26" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>System update (takes time!)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Board IP:</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QComboBox" name="comboBox_2">
          <item>
           <property name="text">
            <string>ON</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OFF</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="21" column="1">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Auto</string>
          </property>
         </widget>
        </item>
        <item row="27" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>Run all tests</string>
          </property>
         </widget>
        </item>
        <item row="23" column="0" colspan="2">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="25" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Pull updates from repository</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Board status:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Session time:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>&lt;p style=&quot;color:red&quot;&gt;DISCONNECTED!&lt;/p&gt;</string>
          </property>
         </widget>
        </item>
        <item row="18" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Sensor fan:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>&lt;h2 align=&quot;center&quot;&gt;CONNECTION&lt;/h2&gt;</string>
          </property>
         </widget>
        </item>
        <item row="19" column="1">
         <widget class="QSlider" name="horizontalSlider_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Processor fan:</string>
          </property>
         </widget>
        </item>
        <item row="30" column="0" colspan="2">
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>&lt;h2 align=&quot;center&quot;&gt;EXIF&lt;/h2&gt;</string>
          </property>
         </widget>
        </item>
        <item row="29" column="0" colspan="2">
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Set...</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>321</width>
        <height>601</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>317</width>
         <height>597</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="15" column="2">
         <widget class="QComboBox" name="comboBox_4">
          <item>
           <property name="text">
            <string>Original</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Low (Fast transfer)</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>Transfer quality:</string>
          </property>
         </widget>
        </item>
        <item row="14" column="2">
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1" colspan="2">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="lineEdit_4"/>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>Save dir:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Exposure (ms):</string>
          </property>
         </widget>
        </item>
        <item row="12" column="2">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>&lt;h2 align=&quot;center&quot;&gt;FILES&lt;/h2&gt;</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>&lt;h2 align=&quot;center&quot;&gt;IMAGING&lt;/h2&gt;</string>
          </property>
         </widget>
        </item>
        <item row="13" column="2">
         <widget class="QLineEdit" name="lineEdit_5">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Interval:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string># Images:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>600</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;p style=&quot;color:yellow&quot;&gt;&lt;strong&gt;Confused? Please read the tutorials.&lt;/strong&gt;&lt;/p&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Capture!</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea_3">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>341</width>
        <height>591</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>337</width>
         <height>587</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>321</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;h2 align=&quot;center&quot;&gt;CAPTURE&lt;/h2&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>550</y>
          <width>321</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>START!</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>221</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;strong&gt;Please review your settings.&lt;/strong&gt;</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>321</width>
          <height>371</height>
         </rect>
        </property>
       </widget>
       <widget class="Line" name="line_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>420</y>
          <width>301</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>460</y>
          <width>181</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_31">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>460</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Imaging:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_32">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>520</y>
          <width>301</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ETA: 0 seconds</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_33">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>480</y>
          <width>59</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Current</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>480</y>
          <width>181</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_34">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>480</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>0/0s</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_35">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>430</y>
          <width>321</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;h2 align=&quot;center&quot;&gt;PROGRESS&lt;/h2&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_36">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>500</y>
          <width>301</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;span style=&quot;color:green&quot;&gt;[CAPTURING]&lt;/span&gt; / [IDLE]</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_37">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>460</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>0/0</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>590</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;p style=&quot;color:yellow&quot;&gt;&lt;strong&gt;Click &lt;span style=&quot;color:green&quot;&gt;START!&lt;/span&gt; to start capturing!&lt;/strong&gt;&lt;/p&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>751</width>
      <height>581</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;p align=&quot;center&quot; style=&quot;color:grey&quot;&gt;&lt;h1&gt;NO OUTPUT&lt;/h1&gt;&lt;/p&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>580</y>
      <width>711</width>
      <height>87</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>580</y>
      <width>33</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>620</y>
      <width>33</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>â†“</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="actionTutorials"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_AstroPi"/>
   </widget>
   <widget class="QMenu" name="menuBoard">
    <property name="title">
     <string>Board</string>
    </property>
    <addaction name="actionPull_updates"/>
    <addaction name="actionSystem_update"/>
    <addaction name="actionRun_tests"/>
    <addaction name="actionConnect_to_shell"/>
   </widget>
   <addaction name="menuBoard"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="statusTip">
    <string/>
   </property>
  </widget>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionTutorials">
   <property name="text">
    <string>Tutorials</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_AstroPi">
   <property name="text">
    <string>About AstroPi</string>
   </property>
  </action>
  <action name="actionPull_updates">
   <property name="text">
    <string>Pull updates</string>
   </property>
  </action>
  <action name="actionSystem_update">
   <property name="text">
    <string>System update</string>
   </property>
  </action>
  <action name="actionRun_tests">
   <property name="text">
    <string>Run tests</string>
   </property>
  </action>
  <action name="actionConnect_to_shell">
   <property name="text">
    <string>Connect to shell</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../assets/AstroPi.png"/>
 </resources>
 <connections/>
</ui>
